demo/main.ts

```diff
@@ -115,7 +115,10 @@ function jsonXmlReplacer(_key: string, value: any): any {
 async function runUpdatingXQuery(script: string) {
 	const result = await fontoxpath.evaluateUpdatingExpression(script, xmlDoc, null, null, {
 		debug: true,
-		disableCache: true
+		disableCache: true,
+		logOutput: m => {
```

Comment:
Let's add a `<pre>` element to the demo page and output the log to there, that we, we even have a good example use case